components/options/about/about.js

```diff
@@ -0,0 +1,49 @@
+/*
+*    StatusPilatus: Monitor your PC like never before!
+*    Copyright (C) 2018 PilatusDevs
+*
+*    This program is free software: you can redistribute it and/or modify
+*    it under the terms of the GNU General Public License as published by
+*    the Free Software Foundation, either version 3 of the License, or
+*    (at your option) any later version.
+*
+*    This program is distributed in the hope that it will be useful,
+*    but WITHOUT ANY WARRANTY; without even the implied warranty of
+*    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*    GNU General Public License for more details.
+*
+*    You should have received a copy of the GNU General Public License
+*    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/* global settings saveSettings */
+"use strict";
+
+const opn = require('opn');
+
+module.exports = {
+    init: initAbout,
+    refresh: refreshAbout,
+    activate: activateAbout
+};
+
+function initAbout() {
+    $("#version").append("Version " + process.env.npm_package_version);
```

Comment:
When I was in a rush I linked you the StackOverflow page where you got this solution, but this is not the right one. As a comment on [that solution](https://stackoverflow.com/a/22339262) says: 

> It works within an electron app started with npm start, but not within a built electron app

The proper solution for our usecase is found [here](https://electronjs.org/docs/api/app#appgetversion).

Reply:
Well time for another push I guess